static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -407,7 +407,9 @@ define([
     }
 
     function shouldRenderLabel($adSlot) {
-        return $adSlot.data('label') !== false && qwery('.ad-slot__label', $adSlot[0]).length === 0;
+        return !$adSlot.hasClass('ad-slot--frame') &&
```

Comment:
I'd be tempted to add a flag to the creative object itself and return that from breakout-iframe, but I'd be a little worried about exposing those objects to other code, because they handle everything through mutable public fields. I guess you could have a static `needsLabel` field on the constructor itself, but it'd be a bit of an odd thing for the breakout script to return.

In any case, though, only the creative template itself knows whether it can handle its own labels, and so that's what should probably control whether labels are added further down the line.